### PR TITLE
Content update

### PR DESCRIPTION
**Pull request by gkindavishal**

### CODE REVIEW

File: config/authors/karunganessh.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Arun Ganessh",
+  "bio": "",
```
Comment (5wathi) [inline config/authors/karunganessh.json:3]: Arun has 4+ years of experience in Developing Android, AR/MR/VR Applications using java, C, C++ language. In addition, he has 1+ years of experience in SAS, Oracle Apex, SQL, HTML, Opengl c++ ,Vulkan. Currently a Lead Engineer at Boeing.

**Review by 5wathi — CHANGES_REQUESTED** [review]

File: config/flutter/blogs/flutter-google-map-with-direction-6a26ad875083.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "expertise": "BEGINNER",
+  "version": 1,
+  "source": "medium",
+  "authorIds": [
+    "shubham"
+  ],
+  "author": "Shubham Narkhede",
+  "title": "Google Map with Direction",
```
[the remark this answers is not in the document]
Comment (gkindavishal) [reply config/flutter/blogs/flutter-google-map-with-direction-6a26ad875083.json:9]: Updated

File: config/authors/aderibigbejesutoni860.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Jesutoni Aderibigbe",
+  "bio": "",
```
[the remark this answers is not in the document]
Comment (gkindavishal) [reply config/authors/aderibigbejesutoni860.json]: Updated

File: config/authors/karunganessh.json (hunk 1 above)
Comment (gkindavishal) [reply config/authors/karunganessh.json]: Updated

**Comment by gkindavishal** [comment]
Updates bio as requested in #640

**Comment by gkindavishal** [comment]
@5wathi please review the updates

File: config/authors/dev-tayy.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "dev-tayy",
```
Comment (5wathi) [inline config/authors/dev-tayy.json:2]: Name: Yusuf Ahmed Tijani

File: config/authors/evd-evanss.json [hunk 5]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "evd-evanss",
```
Comment (5wathi) [inline config/authors/evd-evanss.json:2]: Name: Evandro Costa

File: config/authors/hannydevelop.json [hunk 6]
```diff
@@ -1,6 +1,7 @@
 {
-  "name": "hannydevelop",
+  "name": "Ugochi Ukpai",
   "bio": "",
```
Comment (5wathi) [inline config/authors/hannydevelop.json:3]: I am Ugochi Ukpai, a Marine engineer, technical writer and Rust developer. I love to share knowledge about my transition from marine engineering to software development in the form of writing, to encourage people who love software development.

File: config/authors/idrisadeyemi01.json [hunk 7]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "IdrisAdeyemi01",
```
Comment (5wathi) [inline config/authors/idrisadeyemi01.json:2]: Name: Idris Adeyemi

**Review by 5wathi — CHANGES_REQUESTED** [review]

**Review by 5wathi — COMMENTED** [review]
LGTM @gkindavishal

**Pull request merged**